### PR TITLE
Feature/dns filter

### PR DESCRIPTION
**Pull request by fdurand**
# Description

Added a way to answer custom reply in pfdns with dns filters rules
# Impacts

pfdns
# Delete branch after merge

YES
# NEWS file entries
## New Features
- pfdns filter engine. Added a way to return custom answer in pfdns

### CODE REVIEW

File: lib/pf/ConfigStore/DNSFilters.pm [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package pf::ConfigStore::DNSFilters;
```
Comment (julsemaan) [inline lib/pf/ConfigStore/DNSFilters.pm]: Hate to say these, but the naming is wrong.

Should be DNS_Filters everywhere

Comment (fdurand) [inline lib/pf/ConfigStore/DNSFilters.pm]: yeah i hate when you say that.

Comment (julsemaan) [inline lib/pf/ConfigStore/DNSFilters.pm]: I hate myself even more now that I know you hate it too.

I guess whats left to do is to go cry in my shower alone

File: .gitignore [hunk 2]
```diff
@@ -52,6 +52,10 @@ conf/nessus/remotescan.nessus
 conf/networks.conf
 conf/radiusd/acct.conf
 conf/radiusd/auth.conf
+conf/dns_filters.conf
+conf/radiusd/radiusd.conf
```
Comment (julsemaan) [inline .gitignore]: Does this belong in this PR ?

Comment (fdurand) [inline .gitignore]: Probably because of the rebase

**Comment by julsemaan** [comment]
This is missing packaging

**Pull request merged**